distributor-widget/reference.md

```diff
@@ -104,6 +104,14 @@ distributor.setEndDate("2019-12-18")
 
 Sets a new voucher code value.
 
+#### setAdultCount\(adultCount\)
+
+* `adultCount` Type: `number` - number of adults for pre-selection of room occupancy and price adjustment.
+
```

Comment:
```suggestion
#### setAdultCount\(adultCount\)

* `adultCount` Type: `number` - Number of adults to set 

Sets the number of adults that should be selected by default. 
```
Why:
- For consistency with other parts of this guide, we should: 
  - Describe the value inline. 
  - Use capital at the start of inline description. 
  - Describe more details in own paragraph. 
- I don't think people reading this guide will understand what this is doing. Could we make it clearer to total amateurs? Would new description in suggestion work?


The same applies for `setChildCount`.